admin_guide/quota.adoc

```diff
@@ -149,6 +149,10 @@ A *BestEffort* scope restricts a quota to limit the following resources:
 
 - `*pods*`
 
+*BestEffort* scoped pods can take advantage of slack compute resources in their
+namespace, and then require that users make an explicit resource request for
+pods that require a higher quality of service.
```

Comment:
A BestEffort pod can use slack compute resources that are unused on its scheduled node.

A BestEffort pod is a pod that does not have a compute resource requirement for cpu and memory.